acceptance/tests/apply_plan_ssh.rb

```diff
@@ -94,32 +70,6 @@
     end
   end
 
-  step "puppet service should be stopped" do
```

Comment:
Is this to ensure that `apply_prep` doesn't start the Puppet service? Why was this bit removed?

Reply:
That was a mistake. Added back in.